# Goal
# We always want to command. Don't need this bool
# bool command_laser   # True: Use the command below. False: Use existing profile
float32 start_angle
float32 end_angle
float32 speed
bool hi_fidelity
bool continuous
---
# Result
sensor_msgs/PointCloud2 cloud

---
# Feedback
sensor_msgs/PointCloud2 cloud

